### PR TITLE
replace fake IDs with placeholders

### PR DESCRIPTION
**Pull request by stevemar**
the fake IDs seem to be causing no end of confusion,
so replace them with placeholders instead.

### CODE REVIEW

File: ?
Comment (markstur) [inline ?:1]: "workspance"?

Comment (markstur) [inline ?:1]: typo in staion

Comment (markstur) [inline ?:1]: ditto

Comment (markstur) [inline ?:1]: both staion and spance

**Review by scottdangelo — APPROVED** [review]

**Review by markstur — CHANGES_REQUESTED** [review]
some typos

File: ?
Comment (stevemar) [reply ?:1]: done

Comment (stevemar) [reply ?:1]: done

Comment (stevemar) [reply ?:1]: done

Comment (stevemar) [reply ?:1]: done

**Review by markstur — APPROVED** [review]

**Pull request merged**